src/main/java/core/basesyntax/service/cvs/CsvParseService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.cvs;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface CsvParseService {
+    List<FruitTransaction> parse(List<String> infoFromFile);
```

Comment:
```suggestion
    List<FruitTransaction> parse(List<String> data);
```

As this info could be not only from file